### PR TITLE
Integration Tests

### PR DESCRIPTION
**Pull request by bdwyer2**
Added integration tests using Travis CI and switched to the `codait/max-base` image

### CODE REVIEW

**Comment by lresende** [comment]
How are you planning to use the tests .py? How about just adding a travis.ci build that uses the docker image from the project and in the script, section do a curl?

**Comment by bdwyer2** [comment]
@lresende Thank you for the suggestion. The intent of adding integration testing is so that we can catch problems that occur to other models when we make changes to the base image but testing PR's is a nice benefit.

We will also need to setup something else to test all the models with a new base image before we push a new base image.

File: tests/test.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    assert c.getinfo(pycurl.RESPONSE_CODE) == 200
+    c.close()
+
+    response = b.getvalue()
+    response = json.loads(response)
+
+    assert response['status'] == 'ok'
+    assert response['image_size'] == [513, 256]
+    assert len(response['seg_map']) == response['image_size'][1]
+
```
Comment (frreiss) [inline tests/test.py]: Can you please add a sanity check here to ensure that the returned image segmentation data makes sense? No need to check the entire segmentation map; just verify that, say, three different points in the test image are still labeled with the same label they had when the model was known to be correct.

Comment (bdwyer2) [reply tests/test.py]: Done

**Pull request merged**